paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerTest.kt

```diff
@@ -2368,6 +2365,17 @@ internal class DefaultFlowControllerTest {
         assertThat(arguments.shippingDetails).isNull()
     }
 
+    @Test
+    fun `on init, bootstrap is called`() = confirmationTest {
```

Comment:
Probably not the right name, and I'm not sure if this is testing the right thing.